python/vast/utils/arrow.py

```diff
@@ -0,0 +1,26 @@
+import ipaddress
+
+import pyarrow as pa
+
+
+class IPAddressType(pa.PyExtensionType):
```

Comment:
In VAST this type is called `vast.address`, and its name in code is `address_type`. I think we should name it `AddressType` to match developer expectations here.

Reply:
Since it's called `IPAddress` in the native Python library, I find this name more pythonic.

Reply:
I removed the type.